src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = (int) (Math.random() * 100) + 1;
```

Comment:
Don't use magic numbers, move them to constants. [Reference](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants).